lib/gettext.js

```diff
@@ -7,7 +7,7 @@
     // default values that could be overriden in i18n() construct
     var defaults = {
       domain: 'messages',
-      locale: document.documentElement.getAttribute('lang') || 'en',
+      locale: window && document.documentElement.getAttribute('lang') || 'en',
```

Comment:
Shouldn't we use `(window && document.documentElement.getAttribute('lang')) || 'en'` instead?